packages/serialize/src/index.js

```diff
@@ -290,10 +293,7 @@ function createStringFromObject(
           (registered == null || registered[value[0]] === undefined)
         ) {
           for (let i = 0; i < value.length; i++) {
-            string += `${processStyleName(key)}:${processStyleValue(
-              key,
-              value[i]
-            )};`
+            string += `${key}:${processStyleValue(key, value[i])};`
```

Comment:
Why is processStyleName not being used here?

Reply:
Oh, I was experimenting with code changes - didn't mean to actually commit this. Gonna revert this back in a moment and gonna add a test for this.